runtime/lib/app-runtime.js

```diff
@@ -1149,12 +1135,10 @@ AppRuntime.prototype.login = _.singleton(function login (options) {
     }
 
     // login -> mqtt
-    this.component.custodian.onLogout()
```

Comment:
What states is custodian be while login process is still active?

Reply:
The state is LOGGING_IN.